langserve/server.py

```diff
@@ -866,6 +877,41 @@ async def playground(file_path: str, config_hash: str = "") -> Any:
             file_path,
         )
 
+    @app.post(namespace + "/feedback")
+    async def feedback(feedback_create_req: FeedbackCreateRequest) -> Feedback:
+        """
+        Send feedback on an individual run to langsmith
+        """
+
+        if not tracing_is_enabled():
+            raise HTTPException(
+                400,
+                "The feedback endpoint is only accessible when LangSmith is "
+                + "enabled on your LangServe server.",
+            )
+
+        feedback_from_langsmith = langsmith_client.create_feedback(
```

Comment:
One note here for the reviewers:

When you call this API with a run_id that doesn't exist in your tenant, the API will still succeed. This, I assume, is because LangSmith has eventual consistency on run IDs themselves due to our redis queueing structure. In order to make sure this isn't a security issue where we can annotate runs in different tenants, I did the following:

1. created a run (ID=123 for simplicity) in my personal tenant
2. set my LangServe server to use the LangChain tenant 
3. sent a `POST .../feedback` request to the server using ID=123
4. ensured that the feedback did not get added to my run
5. set my LangServe server to use my personal tenant
6. sent a `POST .../feedback` request to thes erver using ID=123 again
7. ensured that the feedback _did_ get added to my run

It seems this is fine from a security perspective, but from a dev perspective, this is pretty annoying. 

It would be really nice if when we enqueued a parent run, we synchronously wrote a placeholder record postgres (or at least treated redis as a write through cache). This would mean we could check run existence and send back good error messages.

Reply:
This means you can use the feedback endpoint technically to give arbitrary feedback to other runs outside of your current project/runnable as well from within your langsmith tenant

Reply:
Maybe chat with @hinthornw if you have concerns -- I doubt we're going to change this at this stage, but maybe we should do something to document expected behavior better. I was a bit surprised that uuids had to be unique across projects.

Reply:
We talked about it a bit at lunch today. Is there somewhere we track backlog tech debt like this that may or may not become an issue in the future?

Reply:
Github issues is good at this stage since the volume isn't high, and let's other developers see what we're working on